Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -8,8 +8,8 @@
 $ErrorActionPreference = 'Stop'
 
 $toolVersion = @{
-    dbDeploy = "2.0.0-b10031"
-    codeGen = "5.0.0-b10352"
+    dbDeploy = "2.0.0-b10033"
```

Comment:
needs to be `2.0.0`

everything else should be `5.0.0`

Reply:
OK

Reply:
done . Thanks

Reply:
agreed